sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_custom_forms_async.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_custom_forms_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a form from a document with a custom
+    trained model. The form must be of the same type as the forms the custom model
+    was trained on. To learn how to train your own models, look at
+    sample_train_unlabeled_model_async.py and sample_train_labeled_model_async.py
+USAGE:
+    python sample_recognize_custom_forms_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_MODEL_ID - the ID of your custom trained model
+"""
+
+import os
+import asyncio
+from pathlib import Path
+
+
+class RecognizeCustomFormsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    model_id = os.environ["CUSTOM_TRAINED_MODEL_ID"]
```

Comment:
If there's a way to provide this so the sample works out of the box, that would be ideal.

Reply:
I link to samples of how to train your own models in the description. Would it be better to move those to the environment variable description for clarity?

Reply:
Ah, missed that.